Error after update to Angular 8

After updating to Angular 8 build fails with error: **ERROR in node_modules/@ngx-kit/core/src/kit-overlay/kit-overlay.service.d.ts(2,32): error TS2307: Cannot find module '@angular/core/src/di/provider'.**

It's caused by line `import { StaticProvider } from '@angular/core/src/di/provider';`
The reason is that `StaticProvider` type isn't exported anymore.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-kit-src",
-  "version": "7.3.0",
+  "version": "7.3.2",
   "license": "MIT",
   "scripts": {
     "clean": "rimraf ./dist && rimraf ./dist-server && rimraf ./covarage && rimraf ./tmp",

--- a/packages/collection/lib/demo.module.ts
+++ b/packages/collection/lib/demo.module.ts
@@ -1,5 +1,4 @@
-import { NgModule } from '@angular/core';
-import { Type } from '@angular/core/src/type';
+import { NgModule, Type } from '@angular/core';
 import { UiAccordionDemoComponent } from './ui-accordion/demo/ui-accordion-demo.component';
 import { UiAccordionDemoModule } from './ui-accordion/demo/ui-accordion-demo.module';
 import { UiAlertDemoComponent } from './ui-alert/demo/ui-alert-demo.component';

--- a/packages/core/src/kit-overlay/kit-overlay.service.ts
+++ b/packages/core/src/kit-overlay/kit-overlay.service.ts
@@ -10,12 +10,12 @@ import {
   NgZone,
   Optional,
   SkipSelf,
+  StaticProvider,
   TemplateRef,
   Type,
   ViewContainerRef,
   ViewRef,
 } from '@angular/core';
-import { StaticProvider } from '@angular/core/src/di/provider';
 import { Observable, Subject, Subscription } from 'rxjs';
 import { KitPlatformService } from '../kit-platform/kit-platform.service';
 import { KitOverlayComponentRef } from './kit-overlay-component-ref';

